### PR TITLE
XW-1957 | add temporary wikia-logo styles

### PR DESCRIPTION
**Pull request by tomasznapieralski**
https://wikia-inc.atlassian.net/browse/XW-1957
https://github.com/Wikia/app/pull/11265

Styles are taken from existing Oasis global nav

@Wikia/x-wing

### CODE REVIEW

**Comment by rogatty** [comment]
LGTM

File: components/_global-navigation.scss [hunk 1]
```diff
@@ -101,6 +101,30 @@ $notification-counter-size: 18px;
...
+			font-size: 10px;
+			font-weight: bold;
+			line-height: 16px;
+			margin-left: 8px;
+			text-transform: uppercase;
+		}
+
+		svg {
+			fill: $color-dark-blue;
+			height: 100%;
```
Comment (Warkot) [inline components/_global-navigation.scss]: Maybe you could use a value in `px`?

File: components/_global-navigation.scss [hunk 2]
```diff
@@ -101,6 +101,30 @@ $notification-counter-size: 18px;
...
 	}
 
+	// These are temporary styles which should be removed after 4th October 2016
+	&__logo-wikia {
+		flex-basis: 0;
+
+		span {
+			border-bottom: 1px solid;
+			border-top: 1px solid;
+			color: #aaa;
```
Comment (Warkot) [inline components/_global-navigation.scss]: Do we have a variable for that? or maybe something close from Design System?

File: components/_global-navigation.scss [hunk 3]
```diff
@@ -101,6 +101,30 @@ $notification-counter-size: 18px;
...
+			display: inline-block;
+			font-size: 10px;
```
Comment (Warkot) [inline components/_global-navigation.scss]: That's `$typescale-size-minus-3`.

**Comment by Warkot** [comment]
Some minor comments, besides that LGTM. 👍

**Comment by Warkot** [comment]
LGTM

**Comment by kvas-damian** [comment]
LGTM

**Pull request merged**